src/overseer/config.clj

```diff
@@ -3,30 +3,41 @@
   optional, and Overseer should 'just work' out of the box. The entire set
   of configuration options are:
 
-    :store - Required backend store type to connect to. For now, must be :datomic
+    :store - Required backend store type to connect to. One of :datomic, :jdbc
              The corresponding key of the same name must also be present
              to configure the store
 
     :datomic - map configuring the Datomic store (required if using Datomic store)
       :uri - If using Datomic store, required connection String, ex:
              \"datomic:free://localhost:4334/overseer\"
 
-  :sentry - Optional map configuring the Sentry error backend
-    :dsn - String DSN to use
+    :jdbc-adapter - String, one of mysql, h2 (required if using JDBC store).
+                    Used for backend-specific system configuration
 
-  :detector-sleep-time - Optional; How long to sleep between ready job detector runs in ms
-                         (default: 2000)
+    :jdbc - clojure.java.jdbc \"database spec\" (required if using JDBC store)
+            - map of :dbtype, :dbname and other options as needed, such as :user, :password, :host, :port
+            - map of driver :classname, :subprotocol, :hostname, :port, :subname, :user, :password
+            - map of :connection-uri String option passed directly to driver
+            - map of :name and :environment keys for JNDI connection
+            - Can also just be a String JDBC URI
+            See http://clojure-doc.org/articles/ecosystem/java_jdbc/home.html#setting-up-a-data-source
 
-  :sleep-time - Optional; How long to sleep in ms if the job queue is empty (default: 10000)
+    :sentry - Optional map configuring the Sentry error backend
+      :dsn - String DSN to use
 
-  :heartbeat - map of optional attributes to configure worker heartbeating
-    :enabled - When enabled, each node will periodically persist a timestamp
-               'heartbeat' via the DB and also act as a monitor resetting jobs
-               detected to be failing heartbeat checks
-               (default: true)
-    :sleep-time - How long to sleep in ms before persisting heartbeat (per-worker)
-                  (default: 60000)
-    :tolerance - How many heartbeats can fail before job is considered dead
+    :detector-sleep-time - Optional; How long to sleep between ready job detector runs in ms
+                           (default: 2000)
+
+    :sleep-time - Optional; How long to sleep in ms if the job queue is empty (default: 10000)
+
+    :heartbeat - map of optional attributes to configure worker heartbeating
```

Comment:
I suppose that's true; what's missing is theoretically a `if heartbeating is disabled, use old consider-started-eligible logic` switch. I'm not terribly attached to maintaining that and am okay making this mandatory